### PR TITLE
Fix Fx47 compat tests

### PR DESCRIPTION
**Pull request by wagnerand**
r? @EnTeQuAk

### CODE REVIEW

File: tests/compat/test_gecko40.py [hunk 1]
```diff
@@ -46,13 +46,6 @@ def test_extIApplication(self):
         self.assert_silent()
         self.assert_compat_warning(type_='warning')
 
-    def test_Application(self):
```
Comment (wagnerand) [inline tests/compat/test_gecko40.py]: The validation currently does not support multiple checks for the same instance call. But as compat tests for 40 ran a long time ago and won't run again, we should be ok.

File: tests/compat/test_gecko47.py [hunk 2]
```diff
@@ -89,7 +95,11 @@ def test_changed_return_type(self):
         assert self.compat_err.warnings[0]['compatibility_type'] == 'error'
         assert self.compat_err.warnings[0]['message'].startswith(message)
 
-        script = 'let slots = testModule.listSlots();'
+        script = '''
+            var modules = modules.QueryInterface(Components.interfaces.nsIPKCS11Module);
+            modules.listSlots();
+        '''
+        self.run_script_for_compat(script)
```
Comment (wagnerand) [inline tests/compat/test_gecko47.py]: The original patch did not actually run the script.

Comment (EnTeQuAk) [inline tests/compat/test_gecko47.py]: great catch, sadpanda

**Comment by EnTeQuAk** [comment]
looks great, lgtm.

r+

**Comment by wagnerand** [comment]
Thank you.

**Pull request merged**